lib/Dialect/OM/OMOps.cpp

```diff
@@ -74,6 +74,33 @@ static void printPathString(OpAsmPrinter &p, Operation *op, PathAttr path,
   p << '\"';
 }
 
+static ParseResult parseFieldLocs(OpAsmParser &parser, ArrayAttr &fieldLocs) {
+  if (parser.parseOptionalKeyword("field_locs"))
+    return success();
+  uint count = 0;
+  auto parseElt = [&]() -> ParseResult {
+    if (count > 0) {
+      parser.emitError(parser.getCurrentLocation(),
+                       "found more than one field_locs array");
+      return failure();
+    }
+    count++;
```

Comment:
nit: we generally prefer pre-increment: https://llvm.org/docs/CodingStandards.html#prefer-preincrement

Reply:
Also- this parseElt is really just so we can use parseCommaSeparatedList, but we actually just want something surrounded by parentheses right? It seems like we actually error if there are more than 1 value separated by commas, which seems a little weird. Maybe just something like:

```
if (parseLParen() || parseAttribute(fieldLocs) || parseRParen())
```